lighthouse-core/audits/accessibility/accesskeys.js

```diff
@@ -10,8 +10,9 @@
  * See base class in axe-audit.js for audit() implementation.
  */
 
-const AxeAudit = require('./axe-audit.js');
-const i18n = require('../../lib/i18n/i18n.js');
+import AxeAudit from './axe-audit.js';
+
+import * as i18n from '../../lib/i18n/i18n.js';
```

Comment:
In this particular case it was a quick decision made while bulk-editing. It could probably use the named import just fine.

In general, I do sometimes find it hard to know when an import should be used as named or as "namespaced" imports. For example, looks at navigation.js, prepare.js, emulation.js - just to name a few. These are imported as namespaced because it seems to be easier to read? And that was how the code looked before, so it was simpler to transition to in this PR.